depends/packages/univalue.mk

```diff
@@ -0,0 +1,21 @@
+package=univalue
+$(package)_version=1.0.1
+$(package)_download_path=https://codeload.github.com/jgarzik/$(package)/tar.gz
```

Comment:
AFAIK that's just a mirror of @jgarzik 's upstream? Note he could very well add additional maintainers to his upstream repo as well, if he wants.